### PR TITLE
Move TINC tests to Behave

### PR DESCRIPTION
**Pull request by nadeemg**
The tests for gpstate, gpinitstandby and gpactivatestandby have been ported to the behave framework. Also added some unit tests to cover TINC tests for gpstart and gpstop.

### CODE REVIEW

File: gpMgmt/bin/gpactivatestandby [hunk 1]
```diff
@@ -292,16 +292,16 @@ def check_or_start_standby(options):
...
         return False
 
 #-------------------------------------------------------------------------
-def start_master():
+def start_master(options):
     """Starts the master."""
 
     logger.info('Starting standby master database in utility mode...')
-    gp.GpStart.local('Start GPDB', masterOnly=True)
+    gp.NewGpStart.local('Start GPDB', masterOnly=True, masterDirectory=options.master_data_dir)
```
Comment (jmcatamney) [inline gpMgmt/bin/gpactivatestandby]: We don't need to have two different GpStart and NewGpStart classes that do basically the same thing.  Can we find a way to unify the functionality and get rid of one of them?

Comment (jimmyyih) [reply gpMgmt/bin/gpactivatestandby]: Definitely.  It can be done in a different PR to refactor that out.

**Comment by ashwinstar** [comment]
Not removing the actual test code yet ?

File: gpMgmt/bin/gpactivatestandby (hunk 1 above)
Comment (nadeemg) [reply gpMgmt/bin/gpactivatestandby]: I've added a chore for this.

**Pull request merged**